composer.json

```diff
@@ -8,7 +8,8 @@
   ],
   "license": "MIT",
   "require": {
-    "php": ">=7.1"
+    "php": ">=7.1",
+    "ext-curl": "*"
```

Comment:
It's not strictly *required* as you might use other components and not Curl. It can be in `require-dev`, and in `suggest` though!

Reply:
You're right. I will let the ```ext-curl``` be in the ```require-dev``` key.